public/app/features/inspector/InspectDataOptions.tsx

```diff
@@ -129,10 +131,13 @@ export const InspectDataOptions = ({
               {onOptionsChange && (
                 <Field
                   label={t('dashboard.inspect-data.formatted-data-label', 'Formatted data')}
-                  description={t(
-                    'dashboard.inspect-data.formatted-data-description',
-                    'Table data is formatted with options defined in the Field and Override tabs.'
-                  )}
+                  description={
+                    formattedDataDescription ||
+                    t(
```

Comment:
Not sure why we have `dashboard.` i18n keys in `inspector` feature 🤔  probably should be moved out to dashboards at some point.

Reply:
+1. also the translations are very specific to dashboards, something a bit more generic would allow us to not pass any override. something on the line of (but possibly better) `Keep formatting options from panel` maybe?